CMakeLists.txt

```diff
@@ -623,8 +623,27 @@ endif()
 
 # SDL, required for all targets on win32 because of iconv and SDL_SetHint(SDL_TIMER_RESOLUTION, 0)
 if (BUILD_CLIENT OR WIN32)
-    find_package(SDL2 REQUIRED)
+    if(LINUX)
+        find_package(SDL2 QUIET CONFIG)
+        if(SDL2_VERSION)
+            if (SDL2_VERSION VERSION_GREATER_EQUAL "2.16"
+                AND SDL2_VERSION VERSION_LESS_EQUAL "2.20")
+                message(WARNING "SDL ${SDL2_VERSION} between version 2.16 and 2.20 is known to be buggy, see https://github.com/DaemonEngine/Daemon/issues/600")
```

Comment:
I would think this one should be a warning.

LGTM

Reply:
It is, right? Or do i am missing something?

```cmake
message(WARNING […])
```

Reply:
Yes it's right, my bad!